### PR TITLE
Create panel for Postgres collector runtime

### PR DESCRIPTION
**Pull request by ShashankSinha252**
[PMM-10554](https://jira.percona.com/browse/PMM-10554) : Add panel for collector runtime of Postgres exporter

### CODE REVIEW

File: dashboards/Insight/Prometheus_Exporter_Status.json [hunk 1]
```diff
@@ -105,6 +109,10 @@
   "panels": [
     {
       "collapsed": false,
+      "datasource": {
+        "type": "prometheus",
```
Comment (denisok) [inline dashboards/Insight/Prometheus_Exporter_Status.json:113]: ok, uid should be avoided. @anton-bystrov please help with this PR

**Review by denisok — CHANGES_REQUESTED** [review]

File: dashboards/Insight/Prometheus_Exporter_Status.json (hunk 1 above)
Comment (anton-bystrov) [reply dashboards/Insight/Prometheus_Exporter_Status.json]: Done

**Review by anton-bystrov — APPROVED** [review]

File: dashboards/Insight/Prometheus_Exporter_Status.json [hunk 2]
```diff
@@ -4140,18 +4174,26 @@
...
         "x": 0,
-        "y": 104
+        "y": 96
       },
       "id": 1019,
       "panels": [],
+      "targets": [
+        {
+          "datasource": "Metrics",
+          "refId": "A"
```
Comment (denisok) [inline dashboards/Insight/Prometheus_Exporter_Status.json:4189]: not sure about this one. @anton-bystrov could you please check.

Comment (denisok) [reply dashboards/Insight/Prometheus_Exporter_Status.json:4189]: ok, it was in old dashboard as well

**Review by denisok — APPROVED** [review]
OK, all good for me, but with new schema change we need to test upgrades and if it will start in a new PMM container (not when you add the json in grafana).

**Pull request merged**